### PR TITLE
Update main README and examples

### PR DESCRIPTION
**Pull request by ds58**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,47 +10,26 @@ ROS2 messaging for Java. Implements a ROS2 compatible layer with a Java JNI laye
...
-Publish/subscribe API:
-
-`api("us.ihmc:ros2-library:<version>")` Latest:
-[ ![ihmc-ros2-library](https://maven-badges.herokuapp.com/maven-central/us.ihmc/ros2-library/badge.svg?style=plastic) ](https://maven-badges.herokuapp.com/maven-central/us.ihmc/ros2-library)
-
-Generator for `.msg` -> `.java`:
-
-`api("us.ihmc:ros2-msg-to-pubsub-generator:<version>")` Latest: 
-[ ![ros2-msg-to-pubsub-generator](https://maven-badges.herokuapp.com/maven-central/us.ihmc/ros2-msg-to-pubsub-generator/badge.svg?style=plastic) ](https://maven-badges.herokuapp.com/maven-central/us.ihmc/ros2-msg-to-pubsub-generator)
-
```
Comment (ds58) [inline README.md:28]: This all seemed unnecessary to me

File: README.md [hunk 2]
```diff
@@ -12,47 +10,26 @@ ROS2 messaging for Java. Implements a ROS2 compatible layer with a Java JNI laye
...
 
-See `NonRealtimeROS2PublishSubscribeExample.java` and `RealtimeROS2PublishSubscribeExample.java`.
+### Examples
+See `ROS2TalkerExample.java` and `ROS2PublishSubscribeExample.java`.
 
 ## Environment Options
 | Environment Variable                | Description                                                        |
 |-------------------------------------|--------------------------------------------------------------------|
 | ROS_DOMAIN_ID                       | Set the ROS 2 domain ID to use (integer, 0-101)                    |
-| ROS_DISABLE_SHARED_MEMORY_TRANSPORT | Disable shared memory transport if set to `true`                   |
```
Comment (ds58) [inline README.md:44]: This doesn't exist anymore

File: README.md [hunk 3]
```diff
@@ -12,47 +10,26 @@ ROS2 messaging for Java. Implements a ROS2 compatible layer with a Java JNI laye
...
-
 The intermediate .idl files generated by this library are not valid to be used outside IHMC Pub/Sub. 
-
-## License
-
-Apache 2.0
-
-## Maintainer Notes
-
-See [docs/Making a release.md](docs/Making%20a%20release.md)
```
Comment (ds58) [inline README.md:57]: These are unnecessary

File: ros2-library/src/test/java/us/ihmc/ros2/DataTypesTest.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
-package us.ihmc.ros2.example;
```
Comment (ds58) [inline ros2-library/src/test/java/us/ihmc/ros2/DataTypesTest.java:1]: This test was in the example package

File: ros2-library/src/test/java/us/ihmc/ros2/example/NonRealtimeROS2ListenerExample.java
Comment (ds58) [inline ros2-library/src/test/java/us/ihmc/ros2/example/NonRealtimeROS2ListenerExample.java:1]: This example was more or less a duplicate of the other ones

**Review by calvertdw — APPROVED** [review]

**Pull request merged**